build.gradle

```diff
@@ -31,20 +31,22 @@ configurations {
 }
 
 ext {
-  kafkaVersion = "3.5.1"
+  // Even though Kafka Connect 3.7.0 is out, we're staying with 3.6.1 in order to continue
+  // using the third-party Kafka JUnit tool. See https://github.com/mguenther/kafka-junit?tab=readme-ov-file
+  kafkaVersion = "3.6.1"
 }
 
 dependencies {
   compileOnly "org.apache.kafka:connect-api:${kafkaVersion}"
   compileOnly "org.apache.kafka:connect-json:${kafkaVersion}"
   compileOnly "org.apache.kafka:connect-runtime:${kafkaVersion}"
-  compileOnly "org.slf4j:slf4j-api:1.7.36"
+  compileOnly "org.slf4j:slf4j-api:2.0.13"
 
   implementation 'com.marklogic:ml-javaclient-util:4.7.0'
   // Force DHF to use the latest version of ml-app-deployer, which minimizes security vulnerabilities
-  implementation "com.marklogic:ml-app-deployer:4.6.0"
+  implementation "com.marklogic:ml-app-deployer:4.7.0"
 
-  implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.15.2"
+  implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.17.1"
```

Comment:
The Java Client is only on 2.15.3 - stick with that, particularly as I believe Kafka provides an isolated class loader for a plugin so we won't be impacted by whatever Jackson version Kafka is using (not certain about that, but think it's true).